docs/plugins.md

```diff
@@ -46,6 +47,6 @@ def write_sbom(sbom: SBOM, outfile) -> None:
 [project.entry-points."surfactant"]
 sampleplugin = "sampleplugin"
 ```
-From the same folder as your sampleplugin files, run `pip install .` to install your plugin and surfactant will automatically load and use the plugin.
+Run `surfactant plugin install sampleplugin` to install the plugin. Surfactant will automatically load and use the plugin. Alternatively, from the same folder as your sampleplugin files, run `pip install .`.
```

Comment:
Since `surfactant plugin install` is calling pip, this should probably just say to run `surfactant plugin install .` (or the path to the folder on their local filesystem that the plugin is located in) so that pip doesn't try to find a package on PyPI that either doesn't exist yet (or a completely different package someone else created that happens to have the same name).

We could mention uploading their plugin to PyPI for easier sharing and to enable installation using the PyPI package name for the plugin.